spec/index.html

```diff
@@ -6238,16 +6238,203 @@ <h5>LANG</h5>
+                   </tr>
+                 </tbody>
+               </table>
+             </div>
+          </section>
+
+          <section id="func-haslangdir">
+            <h5>hasLANGDIR</h5>
+            <pre class="prototype nohighlight"> <span class="return">xsd:string</span>  <span class="operator">hasLANGDIR</span> (<span class="type">RDF term</span> <span class="name">term</span>)
+            </pre>
+            <p>
+              The function `hasLANGDIR` returns <code>true</code> if the
+              argument is a literal with a language
+              tag. Otherwise, the function returns false.
+            </p>
+            <p>If the argument is a literal, the function is equivalent to
+              testing for the datatype of the literal being
+              `rdf:dirLangString`.
+            </p>
+             <div class="result">
+               <table>
+                 <thead>
+                   <tr>
+                     <th>Expression</th>
+                     <th>Result</th>
+                   </tr>
+                 </thead>
+                 <tbody>
+                   <tr>
+                     <td><code>hasLANGDIR("abc"@en)</code></td>
+                     <td><code>false</code></td>
+                   </tr>
+                   <tr>
+                     <td><code>hasLANGDIR("abc@"en--ltr)</code></td>
+                     <td><code>true</code></td>
+                   </tr>
+                   <tr>
+                     <td><code>hasLANGDIR("تصميم المواقع"@ar--rtl)</code></td>
+                     <td><code>true</code></td>
+                   </tr>
+                   <tr>
+                     <td><code>hasLANGDIR(1)</code></td>
+                     <td><code>false</code></td>
+                   </tr>
+                   <tr>
+                     <td><code>hasLANGDIR(&lt;http://example/&gt;)</code></td>
+                     <td><code>false</code></td>
+                   </tr>
+                 </tbody>
+               </table>
+             </div>
           </section>
+
           <section id="func-datatype">
             <h5>DATATYPE</h5>
             <pre class="prototype nohighlight"> <span class="return"><span class="type IRI">iri</span></span>  <span class="operator">DATATYPE</span> (<span class="type"><span class="type">literal</span></span> <span class="name">literal</span>)
             </pre>
             <p>Returns the <span class="type datatypeIRI">datatype IRI</span> of a
               <code>literal</code>.</p>
-            <p class="note">
-                The datatype of a literal with a <a data-cite="RDF12-CONCEPTS#dfn-language-tag">language tag</a> is <code>rdf:langString</code>.
-            </p>
+            <div class="note">
+              <p>
+                The datatype of a literal with a
+                <a data-cite="RDF12-CONCEPTS#dfn-language-tag">language tag</a> 
+                has a datatype of <code>rdf:langString</code>.
```

Comment:
The sentence already starts with "The datatype  of":
```suggestion
                is <code>rdf:langString</code>.
```